cfgov/unprocessed/apps/owning-a-home/js/explore-rates/index.js

```diff
@@ -1,5 +1,4 @@
 var $ = require( '../../node_modules/jquery' );
-var jQuery = $;
```

Comment:
are you sure this isn't necessary?

Reply:
I converted the references to it to `$`, per https://github.com/cfpb/cfgov-refresh/pull/3830#pullrequestreview-100261276 .